x-pack/plugins/reporting/server/lib/tasks/execute_report.ts

```diff
@@ -194,12 +194,12 @@ export class ExecuteReportTask implements ReportingTask {
       ...doc,
     });
 
-    this.logger.debug(
+    this.logger.info(
       `Claiming ${claimedReport.jobtype} ${report._id} ` +
-        `[_index: ${report._index}]  ` +
```

Comment:
It's important to switch this to `info` so anyone analyzing logs can look at the time difference between when the job was scheduled (placed in the queue) and when it was claimed (started executing). If the user is sending a ton of requests to generate reports, the latest reports may have to wait a long time before they're claimed.